src/LibFreeType.jl

```diff
@@ -15,14 +18,1505 @@ const LONG_MAX = typemax(Clong)
+    bit_set::FT_Raster_BitSet_Func
+    user::Ptr{Cvoid}
+    clip_box::FT_BBox
+end
+
+const FT_Raster_Params = FT_Raster_Params_
+
+# typedef int ( * FT_Raster_NewFunc ) ( void * memory , FT_Raster * raster )
+const FT_Raster_NewFunc = Ptr{Cvoid}
+
+# typedef void ( * FT_Raster_DoneFunc ) ( FT_Raster raster )
+const FT_Raster_DoneFunc = Ptr{Cvoid}
+
+# typedef void ( * FT_Raster_ResetFunc ) ( FT_Raster raster , unsigned char * pool_base , unsigned long pool_size )
+const FT_Raster_ResetFunc = Ptr{Cvoid}
+
+# typedef int ( * FT_Raster_SetModeFunc ) ( FT_Raster raster , unsigned long mode , void * args )
+const FT_Raster_SetModeFunc = Ptr{Cvoid}
+
+# typedef int ( * FT_Raster_RenderFunc ) ( FT_Raster raster , const FT_Raster_Params * params )
+const FT_Raster_RenderFunc = Ptr{Cvoid}
+
+struct FT_Raster_Funcs_
+    glyph_format::FT_Glyph_Format
+    raster_new::FT_Raster_NewFunc
+    raster_reset::FT_Raster_ResetFunc
+    raster_set_mode::FT_Raster_SetModeFunc
+    raster_render::FT_Raster_RenderFunc
+    raster_done::FT_Raster_DoneFunc
+end
+
+const FT_Raster_Funcs = FT_Raster_Funcs_
+
+const FT_Bool = Cuchar
+
+const FT_FWord = Cshort
+
+const FT_UFWord = Cushort
+
+const FT_Char = Int8
+
+const FT_Byte = Cuchar
+
+const FT_Bytes = Ptr{FT_Byte}
+
+const FT_Tag = FT_UInt32
+
+const FT_String = Cchar
+
+const FT_Short = Cshort
+
+const FT_UShort = Cushort
+
+const FT_Int = Cint
+
+const FT_UInt = Cuint
+
+const FT_Long = Clong
+
+const FT_ULong = Culong
+
+const FT_F2Dot14 = Cshort
+
+const FT_F26Dot6 = Clong
+
+const FT_Fixed = Clong
+
+const FT_Error = Cint
+
+const FT_Pointer = Ptr{Cvoid}
+
+const FT_Offset = Csize_t
+
+const FT_PtrDist = Cptrdiff_t
+
+struct FT_UnitVector_
+    x::FT_F2Dot14
+    y::FT_F2Dot14
+end
+
+const FT_UnitVector = FT_UnitVector_
+
+struct FT_Matrix_
+    xx::FT_Fixed
+    xy::FT_Fixed
+    yx::FT_Fixed
+    yy::FT_Fixed
+end
+
+const FT_Matrix = FT_Matrix_
+
+struct FT_Data_
+    pointer::Ptr{FT_Byte}
+    length::FT_Int
+end
+
+const FT_Data = FT_Data_
+
+# typedef void ( * FT_Generic_Finalizer ) ( void * object )
+const FT_Generic_Finalizer = Ptr{Cvoid}
+
+struct FT_Generic_
+    data::Ptr{Cvoid}
+    finalizer::FT_Generic_Finalizer
+end
+
+const FT_Generic = FT_Generic_
+
+mutable struct __JL_FT_ListNodeRec_
+end
+
+function Base.unsafe_load(x::Ptr{__JL_FT_ListNodeRec_})
+    unsafe_load(Ptr{FT_ListNodeRec_}(x))
+end
+
+function Base.getproperty(x::Ptr{__JL_FT_ListNodeRec_}, f::Symbol)
+    getproperty(Ptr{FT_ListNodeRec_}(x), f)
+end
+
+function Base.setproperty!(x::Ptr{__JL_FT_ListNodeRec_}, f::Symbol, v)
+    setproperty!(Ptr{FT_ListNodeRec_}(x), f, v)
+end
+
+const FT_ListNode = Ptr{__JL_FT_ListNodeRec_}
+
+struct FT_ListRec_
+    head::FT_ListNode
+    tail::FT_ListNode
+end
+
+const FT_List = Ptr{FT_ListRec_}
+
+struct FT_ListNodeRec_
+    prev::FT_ListNode
+    next::FT_ListNode
+    data::Ptr{Cvoid}
+end
+
+const FT_ListNodeRec = FT_ListNodeRec_
+
+const FT_ListRec = FT_ListRec_
```

Comment:
Another thing I'd like to point out is that forward-decls are handled in a new way with Clang.jl's new generator.